playwright/masonry/utils/resizeWidth.mjs

```diff
@@ -0,0 +1,24 @@
+// @flow strict
```

Comment:
Are we using this file rn?

Reply:
If not, we will be very soon (in the rest of the suite of tests)